Quick-Start [DA]/pg_cext.md

```diff
@@ -0,0 +1,148 @@
+# External Languages in PostgreSQL:
+External languages in PostgreSQL provide a powerful way to integrate additional programming languages, extending the database's capabilities beyond its native support. Developers can use these languages to implement custom functionality, access external libraries, and leverage their expertise in preferred programming languages.
+
+Using external languages offers several advantages:
+
+1. **Language Flexibility:** Integrate a wide range of languages like Python, Ruby, or Java to perform tasks that may not be as straightforward in SQL.
+
+2. **Code Reusability:** Reuse existing code written in preferred languages, reducing development time and effort.
+
+3. **Performance Optimization:** Choose languages with performance advantages for specific tasks, improving the overall extension efficiency.
```

Comment:
I think these are much more about PLs, but *execution* performance is definitely a big deal for C extensions.